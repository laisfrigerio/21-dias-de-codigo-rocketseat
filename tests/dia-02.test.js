const { JSDOM } = require('jsdom')
const fs = require('fs')
const path = require('path')

const html = fs.readFileSync(path.resolve(__dirname, '../dia-02/index.html'), 'utf8')

let dom
let container

describe('dia 02', () => {
  beforeEach(() => {
    dom = new JSDOM(html, { runScripts: 'dangerously' })
    container = dom.window.document.body
  })

  test('should match the hello world challenge', () => {
    expect(container).toMatchSnapshot();
  });
})