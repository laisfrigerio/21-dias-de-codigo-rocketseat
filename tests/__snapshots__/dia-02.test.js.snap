// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dia 02 should match the hello world challenge 1`] = `
<body
  class="main-container no-toggle"
>
  
  
  <header />
  
  
  <h1>
    
    
    <span>
      |&gt;
    </span>
    
    Hello World
  
  </h1>
  
  
  <script
    src="../assets/js/include-html.js"
  />
  
  
  <script
    src="scripts.js"
  />
  



</body>
`;
